test/models/subnet_test.rb

```diff
@@ -128,4 +128,15 @@ class Subnet::Test < Subnet; end
                                          :element_name => subnet.name}}},
                  smart_variables)
   end
+
+  test "should not save if MTU is not provided" do
+    subnet = FactoryBot.create(:subnet_ipv4)
+    subnet.mtu = ""
+    refute subnet.save
+    assert_match /is not a number/, subnet.errors[:mtu].first
+  end
+
+  test "should have MTU set to 1500 by default" do
+    assert_equal Subnet.new.mtu, 1500
```

Comment:
please swap the arguments, `assert_equal expected, actual`

Should be `assert_equal 1500, Subnet.new.mtu`